src/shared/helpers/urlHelper.ts

```diff
@@ -242,3 +243,11 @@ export const refget = (params: RefgetUrlParams) => {
     config.refgetBaseUrl
   }/sequence/${checksum}?${searchParams.toString()}`;
 };
+
+export const regulationActivityExternalUrl = (
+  species: CommittedItem,
+  featureId: string
+) => {
```

Comment:
instead of positional arguments, let's use a parameters object (`params: { species: CommittedItem, featureId: string; }`)